src/psyclone/tests/core/symbolic_maths_test.py

```diff
@@ -356,6 +356,8 @@ def test_symbolic_math_functions_with_constants(fortran_reader, expressions):
 
 
 @pytest.mark.parametrize("expressions", [("field(1+i)", "field(i+1)"),
+                                         ("lambda", "lambda"),
```

Comment:
Perhaps add to the test description that it could also be a Python Keyword.

Reply:
Done.